src/pages/guideline_ios/index.js

```diff
@@ -0,0 +1,233 @@
+import React from 'react';
+import Flex from '../../components/Flex';
+import Layout from '../../components/Layout';
+import LeftNav from '../../components/LeftNav';
+import designImg from './../../img/design.png';
+import { Link, graphql } from 'gatsby';
+import ResourcesList from '../../components/ResourceList/ResourcesList';
+import Tabs from '../../components/Tabs';
+import { css } from '@emotion/core';
+
+
+
+export default class GuidelineIosIndexPage extends React.Component {
+  render() {
+    const { data, location } = this.props;
+    const frontmatter = data.ios.edges[0].node.frontmatter;
+    const posts = data.posts.edges;
+
+    return (
+      <Layout location={location}>
+        <Flex
+          direction='row'
+          shrink='0'
+          grow='1'
+          overflow='auto'
+          valign='stretch'
+          css={{
+            width: '100%',
+          }}
+        >
+          <LeftNav title="Fiori For Web" open="false" data={data.leftNav} />
+          <div
+            css={{
+              width: '100%'
+            }}
+          >
+            <div
+              css={{
+                width: '100%',
+                backgroundColor: '#f8f9fb',
+                height: '682px',
+                backgroundImage: 'url(' + designImg + ')',
```

Comment:
@ckoutsiaris I reckon on a large screen the hero banner should still be centre align to the grid, not right aligned.

Reply:
I copy-pasted from existing. Will change it thanks